Frameworks/CoreFoundation/Preferences.subproj/CFPreferences.c

```diff
@@ -190,10 +190,10 @@ if (completePath) {
     CFStringAppend(completePath, CFSTR("Preferences\\"));
     url = CFURLCreateWithFileSystemPath(alloc, completePath, kCFURLWindowsPathStyle, true);
 
-	char cPath[CFMaxPathSize];
-	if (CFURLGetFileSystemRepresentation(url, true, (unsigned char *)cPath, CFMaxPathSize)) {
-		success = _CFCreateDirectory(cPath);
-	}
+    char cPath[CFMaxPathSize];
```

Comment:
Not 100% sure we should change this indentation unless it's in code WinObjC added.